go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/container-storage-interface/spec v1.3.0
 	github.com/golang/protobuf v1.4.3
 	github.com/kubernetes-csi/csi-lib-utils v0.7.0
-	github.com/kubernetes-csi/csi-proxy/client v0.2.2
+	github.com/kubernetes-csi/csi-proxy/client v1.0.0-rc.1
```

Comment:
hey @andyzhangx, FYI we've bumped the rc to v1.0.0, there are no changes in the API and it's just a change in go.mod

Reply:
thanks, I will make changes later.